src/pages/tug/apply/form/form.data.js

```diff
@@ -0,0 +1,128 @@
+import * as Yup from 'yup';
+import { api, getFormData } from '@tidb-community/datasource';
+import { utils } from '@tidb-community/common';
+
+const { buildInitialValues, buildSchema } = utils.form;
+const formData = getFormData();
+const { stageOfCompanyUseOfTidb, preferredWayOfSharing, rolesWantToPlay } = formData.tug.enums;
+const { personalPositions } = formData.org.enums;
+
+export const fetchOrganizationOptions = (word) =>
+  api.orgs.searchCompany({ word }).then((result) =>
+    result.data.map(({ name }) => ({
+      label: name,
+      value: name,
+    }))
+  );
+
+const data = {
+  form: {
+    realName: {
+      name: 'real_name',
+      placeholder: '真实姓名',
+      validator: Yup.string()
+        .min(2, ({ min }) => `真实姓名最短为${min}个字符`)
+        .max(20, ({ max }) => `真实姓名最长为${max}个字符`)
+        .required('真实姓名不可为空'),
+      initialValue: '[empty]',
```

Comment:
Do we have to rely on this 'magic value'?

Reply:
it is only for testing the mockAPI